Samples/Xamarin/HelloWorld/HelloWorld.Ninject/HelloWorld.Ninject/App.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Xamarin.Forms;
+
+namespace HelloWorld
+{
+    public class App : Application
+    {
+        public App()
+        {
+            Bootstrapper bs = new Bootstrapper();
```

Comment:
Everything above this is just standard File > New > Xamarin Forms App. You can start code reviewing here :smile: